app/core/core-file-upload/src/main/kotlin/com/hedvig/android/core/fileupload/DownloadPdfUseCase.kt

```diff
@@ -35,26 +34,26 @@ internal class DownloadPdfUseCaseImpl(
 ) : DownloadPdfUseCase {
   override suspend fun invoke(url: String): Either<ErrorMessage, File> = withContext(Dispatchers.IO) {
     either {
-      val request = Request.Builder()
-        .url(url)
-        .build()
+      try {
+        val request = Request.Builder()
+          .url(url)
+          .build()
 
-      val now = DateTimeFormatter.ISO_DATE_TIME.format(
-        clock.now()
-          .toLocalDateTime(TimeZone.UTC)
-          .toJavaLocalDateTime(),
-      )
+        val now = DateTimeFormatter.ISO_DATE_TIME.format(
+          clock.now()
+            .toLocalDateTime(TimeZone.UTC)
+            .toJavaLocalDateTime(),
+        )
 
-      val downloadedFile = File(context.filesDir, FILE_NAME + now + FILE_EXT)
+        val downloadedFile = File(context.filesDir, FILE_NAME + now + FILE_EXT)
 
-      try {
         val response = okHttpClient.newCall(request).await()
         downloadedFile.sink().buffer().use { fileSink ->
           fileSink.writeAll(response.body!!.source())
         }
         downloadedFile
-      } catch (exception: IOException) {
-        logcat(LogPriority.ERROR, exception) { "Could not download pdf" }
+      } catch (exception: Exception) {
```

Comment:
we were having a crush here because for some quotes the urls that we're getting in staging now are something like "MISSING TRANSLATIONS url for terms and conditions", which does not start with http:// and just crash immediately:)